src/libman/Resources/Text.resx

```diff
@@ -132,6 +132,9 @@
   <data name="DefaultProviderOptionDesc" xml:space="preserve">
     <value>The provider to use if no provider is defined for a given library. (eg. “cdnjs”, “filesystem”)</value>
   </data>
+  <data name="NonInteractiveConsoleMessage" xml:space="preserve">
+    <value>Cannot start an interactive session. Please specify all required arguments on the commandline.</value>
```

Comment:
>interactive session [](start = 27, length = 19)

The phrase "interactive session" seems too technical. Maybe something like, "Unable to read input in this console" or something that explains what the problem is from the user's perspective.

Reply:
Changed the messaging to be: 
```
libman init
libman : Cannot prompt for user input in this console.
Please specify 'DefaultProvider' using the '--default-provider' option.
```

The field name and option string are provided by the caller.